NorthstarDLL/masterserver.cpp

```diff
@@ -231,9 +235,10 @@ void MasterServerManager::RequestServerList()
 					if (!serverObj.HasMember("id") || !serverObj["id"].IsString() || !serverObj.HasMember("name") ||
 						!serverObj["name"].IsString() || !serverObj.HasMember("description") || !serverObj["description"].IsString() ||
 						!serverObj.HasMember("map") || !serverObj["map"].IsString() || !serverObj.HasMember("playlist") ||
-						!serverObj["playlist"].IsString() || !serverObj.HasMember("playerCount") || !serverObj["playerCount"].IsNumber() ||
-						!serverObj.HasMember("maxPlayers") || !serverObj["maxPlayers"].IsNumber() || !serverObj.HasMember("hasPassword") ||
-						!serverObj["hasPassword"].IsBool() || !serverObj.HasMember("modInfo") || !serverObj["modInfo"].HasMember("Mods") ||
+						!serverObj["playlist"].IsString() || !serverObj.HasMember("region") || !serverObj["region"].IsString() ||
```

Comment:
As region fields are optional per @pg9182's mention, this check should not be in here.

The thing is, I don't know how to properly parse the JSON later so that a non-existing JSON field results in a empty string for that line in the serverbrowser

Reply:
Either you can have the region field simply never not exist, and have Atlas send some sort of value that signifies null (empty string, etc.).

Or you can just do something like:
`region = serverObj.HasMember("region) ? serverObj["region"].GetString() : "";`
To set an empty string if `region` is not present

Reply:
oh yeah, that should work ^^

Imma check once pg updated Atlas in regards to no sending region for certain servers ^^

Reply:
@GeckoEidechse?